app/views/tag/index.html.erb

```diff
@@ -21,8 +21,12 @@
 
   <table class="table">
     <tr>
-      <th><a href = "<%= tags_path %>?sort=name"><%= t('tag.index.tag') %></a> <i class="fa fa-arrows-v"></i></th>
-      <th><a href = "<%= tags_path %>?sort=uses"><%= t('tag.index.number_of_uses') %></a> <i class="fa fa-arrows-v"></i></th>
+      <% order_type = "desc" %>
```

Comment:
Can we set `order_type` in the controller like we're setting `@tags`?  I think the controller would be a better place to do that.  In general, from my understanding, logic should be kept out as much as possible from the ERB template.  It should primarily stay in the controller and model.